map/pom.xml

```diff
@@ -62,10 +74,48 @@
             <artifactId>jmock-legacy</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-shapefile</artifactId>
+            <version>15.2</version>
```

Comment:
Can you put the dependency versions in the top level pom to be consistent with the rest of the project?

Reply:
9921bc46eaef1f2b022379c9e24cba52db5b1735